vm/actor/src/builtin/miner/deadline_assignment.rs

```diff
@@ -0,0 +1,185 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use super::{Deadline, SectorOnChainInfo};
+use std::{cmp::Ordering, collections::BinaryHeap};
+
+fn div_rounding_up(dividend: u64, divisor: u64) -> u64 {
```

Comment:
maybe worth tagging this function with `#[inline]`? Definitely optional

Reply:
I think `#[inline]` only matters between crate boundaries, here the function is called in the same crate (in the same file even) so the compiler is already free to inline it. Unless you're talking about force-inlining it?

Reply:
ah, honestly didn't know the compiler could do this automatically